python-scripts/src/main/python/dataproducts/services/consumption/dqp_merger.py

```diff
@@ -0,0 +1,120 @@
+import json
+import pdb
+import os
+import pandas as pd
+
+from datetime import datetime, timedelta, date
+from pathlib import Path
+from azure.common import AzureMissingResourceHttpError
+
+from dataproducts.util.utils import create_json, get_dqp_data_from_blob, \
+    post_data_to_blob, push_metric_event, get_data_from_blob
+
+class DQPMerger:
+    def __init__(self, report_config):
+        self.report_config = report_config
+        self.base_path = None
+
+
+    def drop_stale(self, report_df_, delta_df_):
+        dims = self.report_config['merge']['dims']
+        query = []
+
+        try:
+            grouped_data = delta_df_[dims].drop_duplicates()
```

Comment:
What is this variable doing here?

Reply:
@SanthoshVasabhaktula : There was an issue, and now it is fixed.